.gitignore

```diff
@@ -15,3 +15,4 @@ Gemfile.lock
 spec/elastic_apm.log
 spec/ruby-agent-junit.xml
 .gem
+*.bulk
```

Comment:
What's this?

Reply:
That's a file auto-generated when running the benchmarking locally:
- https://github.com/elastic/apm-agent-ruby/pull/403/files#diff-964990e4e95f38885151096be29638e5R39